pydata_sphinx_theme/layout.html

```diff
@@ -61,7 +66,7 @@
           {% endblock %}
 
           {% block docs_main %}
-          <main class="col-12 col-md-9 col-xl-7 py-md-5 pl-md-5 pr-md-4 bd-content" role="main">
+          <main class="col py-md-5 pl-md-5 pr-md-4 bd-content" role="main">
```

Comment:
setting the main content to just `col` as I believe this will make it grow to fill up available space around it, rather than explicitly defining its size.

Reply:
It seems that this change is causing the problem on https://pydata-sphinx-theme--263.org.readthedocs.build/en/263/demo/demo.html 
I suppose there is some content on that page that is larger, and causes the main text column to be wider if it is not constrained to a max size, and then causing the right toc sidebar to have no space anymore to appear on the side